README.md

```diff
@@ -1,20 +1,78 @@
-# handling-peer-dependencies
+# How to handle peer dependencies when developing modules 
 
-`yarn` and `npm` doesn't provide tools to install peer dependencies for your development environment. There are an opened issue on yarn repository since [27 October 2016](https://github.com/yarnpkg/yarn/issues/1503).
+## What is a peer dependency and what is the problem with it ?
 
-## Steps
+[Peer dependencies](https://nodejs.org/es/blog/npm/peer-dependencies/#the-solution-peer-dependencies) is a specific kind of dependencies really useful for reusable modules. It lets you ask user to install a dependency your module needs to work. It prevents having multiple version of a same module in user's app node_modules. It also have other benefits like 
+lighter javascript files to load on browser side particularly useful for mobile users.
 
-- Install in dependencies in lib `yarn` it also install peerDeps thanks `prepare` script and `install-peers`
-- Build lib `cd lib && yarn build`
-- Create the link of the lib `yarn link`
-- Link the lib in the host app `cd app && yarn link lib`
-- Start app it should works
+The problem with peer dependencies is `npm` and `yarn` doesn't install them at all. This is the right behavior for production purpose but when you are developping you need to test your module in an host app for example. `npm` and `yarn` provide a command to acheive it `link` that basically create a symlink into the host app node_modules to your module source folder. It works fine while using your module in the host app but you also need to execute tasks in your module that needs these dependencies like tests since the dependencies aren't present in your module's node_module folder you will have errors like this:
```

Comment:
`don't`

Reply:
`achieve it called link`

Reply:
`creates`

Reply:
La dernière phrase est trop longue. La revoir pour mieux la découper / simplifier.